invenio_communities/assets/semantic-ui/js/invenio_communities/members/invitations/invitationsModal/MemberSearchBar.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { List, Dropdown } from "semantic-ui-react";
+import { i18next } from "@translations/invenio_communities/i18next";
+import { Image } from "react-invenio-forms";
+import _isEmpty from "lodash/isEmpty";
+import { usersList } from "./mockedData";
+
+export class MembersSearchBar extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isFetching: false,
+      error: false,
+    };
+  }
+
+  serializePeople = (people) => {
```

Comment:
isn't it the same code as above? maybe it make sense reuse it instead of duplicating?

Reply:
Yes, thanks! one of them was a leftover